opentelemetry-sdk/tests/metrics/test_metric_reader_storage.py

```diff
@@ -221,3 +230,462 @@ def test_drop_aggregation(self):
         self.assertEqual(
             [], metric_reader_storage.collect(AggregationTemporality.DELTA)
         )
+
+    def test_conflicting_view_configuration(self):
```

Comment:
I think these would be much more readable added as "integration tests" of the SDK, expressed in terms of the public API. That would also prevent having to change the tests when changes are made to the implementation.

For example https://github.com/open-telemetry/opentelemetry-python/blob/739760586ade075b18d29fa58ce7ac3dbae90869/opentelemetry-sdk/tests/metrics/integration_test/test_disable_default_views.py#L24-L36

Reply:
Sure, I'll add them in a separate PR :v: